templates/js/ai-bot/env/.env.local.user

```diff
@@ -2,4 +2,7 @@
 
 # If you're adding a secret value, add SECRET_ prefix to the name so Teams Toolkit can handle them properly
 # Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
-SECRET_BOT_PASSWORD=
\ No newline at end of file
+SECRET_BOT_PASSWORD=
+SECRET_OPENAI_API_KEY=
+SECRET_AZURE_OPENAI_API_KEY=
+SECRET_AZURE_OPENAI_ENDPOINT=
```

Comment:
Do we need to remove those values from `.localConfigs` file?

Reply:
I think not, these values will still be written to `.localConfigs` during F5.